config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -86,7 +86,7 @@ periodics:
     fork-per-release-generic-suffix: "true"
     testgrid-dashboards: sig-release-master-blocking
     testgrid-tab-name: build-master
-    testgrid-alert-email: "kuberentes-release-team@googlegroups.com"
```

Comment:
kuberentes forever!

Reply:
Some Fejtas just want to watch the world burn 😜